README.md

```diff
@@ -42,14 +45,10 @@ pip install --extra-index-url="https://packages.dea.ga.gov.au" \
   odc-dscache
 ```
 
-**NOTE**: on Ubuntu 18.04 the default `pip` version is awfully old and does not
-support `--extra-index-url` command line option, so make sure to upgrade `pip`
-first: `pip3 install --upgrade pip`.
-
 For Conda Users
 ---------------
 
-Currently there are no `odc-tools` conda packages. But majority of `odc-tools`
+`odc-tools` aren't available as conda packages, but the most
```

Comment:
`odc-apps-ds-tools`, `odc-algo`, `odc-io`, and `odc-cloud` are all available on conda-forge, as are the now separate `odc-stac`, `odc-stats`, and `odc-dscache`